roles/ceph-common/templates/ceph.conf.j2

```diff
@@ -120,7 +120,7 @@ keyring = /var/lib/ceph/radosgw/{{ cluster }}-rgw.{{ hostvars[host]['ansible_hos
 rgw socket path = /tmp/radosgw-{{ hostvars[host]['ansible_hostname'] }}.sock
 log file = /var/log/ceph/{{ cluster }}-rgw-{{ hostvars[host]['ansible_hostname'] }}.log
 rgw data = /var/lib/ceph/radosgw/{{ cluster }}-rgw.{{ hostvars[host]['ansible_hostname'] }}
-rgw frontends = civetweb {{ radosgw_civetweb_options }}
+rgw frontends = civetweb port={{ hostvars[host]['ansible_default_ipv4']['address'] }}:{{ radosgw_civetweb_port }} {{ radosgw_civetweb_options }}
```

Comment:
@SirishaGuduru it's better to use `hostvars[host]['ansible_' + {{ monitor_interface }}]['ipv4']['address']` but as @leseb said, we must handle ipv6 use case so it might change a bit the approach. I'm going to take a deeper look at this PR to find the best way to integrate it.